src/app/contextBroker/contextBroker.cpp

```diff
@@ -1083,7 +1083,6 @@ void orionExit(int code, const std::string& reason)
     destroyAllOntimeIntervalThreads(dbs[ix]);
   }
 
-  mongoDisconnect();
```

Comment:
Maybe is a good idea to revert commit https://github.com/telefonicaid/fiware-orion/commit/7eec274497c49dd094df02f8717d0c87636244a9 (I can tell you how to do it if you don't know), then use string replacement in the IDE:

`mongoDisconnect();` -> `setMongoConnectionForUnitTest(NULL);`

Only for files under test/unittes. In src/, just removing the function is ok.